.github/workflows/latest-release.yml

```diff
@@ -57,3 +57,38 @@ jobs:
       - name: Push discovery engine images to Docker
         run: |
             ./src/build/push_knoxautopolicy.sh ${{ steps.vars.outputs.tag }}
+
+  push-stable-version:
+    name: Create discovery engine Release - 18.04
```

Comment:
Please add a `needs: build` so that this job runs only after the `build` is done.

Reply:
Perfect! Thanks for the point.